metagraph/src/annotation/row_diff_builder.cpp

```diff
@@ -1003,32 +931,24 @@ void convert_batch_to_row_diff(const std::string &pred_succ_fprefix,
                     uint64_t succ_value = is_anchor ? 0 : get_value(source_col, source_idx, j, *succ);
                     if (succ_value != curr_value) {
                         // no reduction, we must keep the bit
-                        auto &v = set_rows_fwd[source_idx][j];
                         if constexpr(with_values) {
-                            v.emplace_back(row_idx, curr_value - succ_value);
+                            set_rows[source_idx][j].emplace_back(row_idx, curr_value - succ_value);
```

Comment:
Before you had `set_rows_fwd` and `set_rows_bwd`, but now it looks like these have been consolidated into a single `set_rows` vector. are they now interleaved or something? I wanted to double-check if this is what you intended.

Reply:
Previously, one of them was storing positions that are already sorted, and the sorting was applied only to the other buffer. But now I figured it's not a big difference if we just sort everything together so now I push everything into the same buffer.